File: WhaleModels/Ranch.cs
using System;

namespace WhaleModels
{
    public class Ranch
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int RancherID { get; set; }
        public Rancher Owner { get; set; }
        public ICollection<Paddock> Paddocks { get; set; }
    }
}

File: WhaleAngular/src/app/models/ranch.ts
import {Facility} from "./facility"
import {Paddock} from "./paddock"
import {Rancher} from "./rancher";
export interface Ranch {
    id: number
    name: string,
    rancherID: number,
    owner: Rancher,
    facilities: Facility[],
    paddocks: Paddock[]
}